src/main/python/camdkit/model.py

```diff
@@ -52,55 +53,669 @@ class LensSerialNumber(StringParameter):
   sampling = Sampling.STATIC
   units = None
+  section = "camera"
 
 class CameraMake(StringParameter):
   """Make of the camera"""
 
   canonical_name = "cameraMake"
   sampling = Sampling.STATIC
   units = None
+  section = "camera"
 
 class CameraModel(StringParameter):
   """Model of the camera"""
 
   canonical_name = "cameraModel"
   sampling = Sampling.STATIC
   units = None
+  section = "camera"
 
 class CameraFirmware(StringParameter):
   """Version identifier for the firmware of the camera"""
 
   canonical_name = "cameraFirmwareVersion"
   sampling = Sampling.STATIC
   units = None
+  section = "camera"
+
+class DeviceSerialNumber(StringParameter):
+  """Unique identifier of the device producing data"""
+
+  canonical_name = "deviceSerialNumber"
+  sampling = Sampling.STATIC
+  units = None
+  section = "device"
+
+class DeviceMake(StringParameter):
+  """Make of the device producing data"""
+
+  canonical_name = "deviceMake"
+  sampling = Sampling.STATIC
+  units = None
+  section = "device"
+
+class DeviceModel(StringParameter):
+  """Model of the device producing data"""
+
+  canonical_name = "deviceModel"
+  sampling = Sampling.STATIC
+  units = None
+  section = "device"
+
+class DeviceFirmware(StringParameter):
+  """Version identifier for the firmware of the device producing data"""
+
+  canonical_name = "deviceFirmwareVersion"
+  sampling = Sampling.STATIC
+  units = None
+  section = "device"
+
+class AnamorphicSqueeze(StrictlyPositiveIntegerParameter):
+  """Nominal ratio of height to width of the image of an axis-aligned square
+  captured by the camera sensor. It can be used to de-squeeze images but is not
+  however an exact number over the entire captured area due to a lens' intrinsic
+  analog nature."""
+
+  canonical_name = "anamorphicSqueeze"
+  sampling = Sampling.STATIC
+  units = "0.01 unit"
+  section = "camera"
+
+class FDLLink(UUIDURNParameter):
+  """Unique identifier of the FDL used by the camera."""
+
+  canonical_name = "fdlLink"
+  sampling = Sampling.STATIC
+  units = None
+  section = "camera"
+
+class ShutterAngle(Parameter):
+  """Shutter speed as a fraction of the capture frame rate. The shutter speed
+  (in units of 1/s) is equal to the value of the parameter divided by 360 times
+  the capture frame rate."""
+
+  canonical_name = "shutterAngle"
+  sampling = Sampling.STATIC
+  units = "degrees (angular)"
+  section = "camera"
+
+  @staticmethod
+  def validate(value) -> bool:
+    """The parameter shall be an integer in the range (0..360000]."""
+
+    return isinstance(value, numbers.Integral) and 0 < value <= 360000
+
+  @staticmethod
+  def to_json(value: typing.Any) -> typing.Any:
+    return value
+
+  @staticmethod
+  def from_json(value: typing.Any) -> typing.Any:
+    return int(value)
+
+  @staticmethod
+  def make_json_schema() -> dict:
+    return {
+      "type": "integer",
+      "minimum": 1,
+      "maximum": 360000
+    }
+  
+class PacketId(UUIDURNParameter):
```

Comment:
This is probably the id of the sample, right? I think the distinction between sample and packet is important since the transport might fragment a sample across multiple packets, or aggregate multiple samples in one packet.